src/components/Modal/BaseModal.js

```diff
@@ -127,9 +127,7 @@ class BaseModal extends PureComponent {
                 hasBackdrop={this.props.fullscreen}
                 coverScreen={this.props.fullscreen}
                 style={modalStyle}
-                // When `statusBarTranslucent` is true on Android, the modal fully covers the status bar.
-                // Since `windowHeight` doesn't include status bar height, it should be added in the `deviceHeight` calculation.
-                deviceHeight={this.props.windowHeight + ((this.props.statusBarTranslucent && StatusBar.currentHeight) || 0)}
+                deviceHeight={this.props.windowHeight}
```

Comment:
I think it should be fine since our device height now always include this extra height from here https://github.com/Expensify/App/pull/15778/files#diff-8c3a1abc2b9785aae309526470e872e7f87b5a7af54cd87d79897b7daf0811b6R3. But yes please let me know if that works, I don’t think I tested this case, only the pdf attachment one